core/src/main/java/nl/nn/adapterframework/configuration/IbisContext.java

```diff
@@ -34,6 +34,7 @@
 import nl.nn.adapterframework.extensions.graphviz.GraphvizEngine;
 import nl.nn.adapterframework.http.RestServiceDispatcher;
 import nl.nn.adapterframework.jdbc.migration.Migrator;
+import nl.nn.adapterframework.lifecycle.IbisApplicationContext;
```

Comment:
Als dit de enige change in deze file is, is hij vast niet nodig

Reply:
Jawel want hij is verplaatst naar een andere package